src/UI/Component/Listing/DescriptiveList.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+
+namespace ILIAS\UI\Component\Listing;
+
+/**
+ * Interface DescriptiveList
+ * @package ILIAS\UI\Component\Listing
+ */
+interface DescriptiveList extends \ILIAS\UI\Component\Component {
```

Comment:
Why not just Descriptive? We already know it's a listing by the namespace, right? I also already followed this Pattern at the Button, where the final class has the unqualified Standard and not StandardButton. We should at least be consistent with this.